### PR TITLE
Select pulse primitive library

### PR DESCRIPTION
**Pull request by dieris**
Choose between standard and diatomic pulses in the config.json as:
```
{...
"PulsePrimitivesLibrary": "standard" / "all90"
...}
```

@blakejohnson not sure if this is what you had in mind, please take a look

### CODE REVIEW

**Comment by dieris** [comment]
Ops, I forgot that X pulses are used to build other primitives

File: QGL/PulsePrimitives/all90_primitives.py [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+@_memoize
+def Y(qubit, **kwargs):
+    return Y90(qubit, **kwargs) + Y90(qubit, **kwargs)
+
+@_memoize
+def Ym(qubit, **kwargs):
+    return Y90m(qubit, **kwargs) + Y90m(qubit, **kwargs)
+
+def AC(qubit, cliffNum, Xonly = False):
```
Comment (blakejohnson) [inline QGL/PulsePrimitives/all90_primitives.py]: They aren't really "atomic Cliffords" if we are composing them of multiple pulses.

**Comment by blakejohnson** [comment]
I posted an alternative to `feature/pulseLibraries-brj`.

**Comment by dieris** [comment]
sure, your alternative is cleaner. Also, I'll leave AC and DiAC separate

**Comment by dieris** [comment]
Replaced by https://github.com/BBN-Q/QGL/pull/81

**Pull request closed without merging**